src/components/Squiggle/index.js

```diff
@@ -14,13 +14,11 @@ const Squiggle = ({ color, repeats }) => (
         text-align: center;
         height: 50px;
         transform: rotate(180deg);
-        background: 0 center repeat-x
-          url('data:image/svg+xml;utf-8,<?xml version="1.0" encoding="UTF-8"?>
-        <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="${SQUIGGLE_WIDTH}px" height="${SQUIGGLE_HEIGHT}px" viewBox="0 0 9 6" enable-background="new 0 0 9 6" xml:space="preserve">
-            <polygon stroke="${encodeURIComponent(
-              color
-            )}" points="4.5,4.5 0,0 0,1.208 4.5,5.708 9,1.208 9,0 "/>
-        </svg>');
+        background-position-x: 0;
+        background-position-y: center;
+        background-repeat: repeat-x;
+        background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' width='100%25' height='8px' viewBox='0 0 9 6' enable-background='new 0 0 9 6' xml:space='preserve'%3e%3cpolygon stroke='%23fb37f1' points='4.5,4.5 0,0 0,1.208 4.5,5.708 9,1.208 9,0 '/%3e%3c/svg%3e");
```

Comment:
Is `width='100%25'` right? If there was no problem with ` SQUIGGLE_HEIGHT ` being used as a variable in IE then we should keep using it, otherwise it should be removed.

Reply:
yes right **width='100%25**' i tried to play around with the width but it doesnt work if i remove it. I tried on production site using inspect element in IE (as site is not working in development enviorement on IE)

Reply:
i checked it again and i think we should remove width property because it is working finr without giving width too ...

Reply:
@SarikaWirtz Can you add the SQUIGGLE_HEIGHT var back in if IE11 can cope with it? Currently it's unused.

Reply:
@nick okay